packages/web-service/src/services/virus-scan.js

```diff
@@ -0,0 +1,45 @@
+import 'dotenv/config'
+import NodeClam from 'clamscan'
+import * as fs from 'fs'
+
+const options = {
+  clamdscan: {
+    socket: null,
+    host: process.env.CS_HOST,
+    port: process.env.CS_PORT,
+    timeout: parseInt(process.env.CS_TIMEOUT),
+    multiscan: false,
+    active: true
+  },
+  preference: 'clamdscan'
+}
+const ClamScan = new NodeClam().init(options)
+
+export async function scanFile (filename) {
+  const dir = `.${process.env.SCANDIR}/${filename}`
```

Comment:
I still think this should be an absolute path - do you want to have a chat about it?